packages/xo-server/src/api/pool.mjs

```diff
@@ -132,7 +132,17 @@ installPatches.description = 'Install patches on hosts'
 // -------------------------------------------------------------------
 
 export async function rollingUpdate({ pool }) {
+  const loadBalancerPlugin = (await this.getPlugins()).find(plugin => plugin.name === 'load-balancer')
```

Comment:
I feel like we keep repeating this pattern, should we had a `this.getPlugin(name)`?